src/documents/fr/idCard/idCardV1.ts

```diff
@@ -1,3 +1,97 @@
-import { Document } from "../../document";
+import { Document, DocumentConstructorProps } from "../../document";
+import { Field, DateField, BaseField } from "../../../fields";
 
-export class IdCardV1 extends Document {}
+export class IdCardV1 extends Document {
+  authority: Field;
+  documentSide: BaseField;
+  idNumber: Field;
+  birthDate: DateField;
+  expiryDate: DateField;
+  birthPlace: Field;
+  gender: Field;
+  mrz1: Field;
+  mrz2: Field;
+  surname: Field;
+  givenNames: Field[] = [];
```

Comment:
all field properties should be documented, take a look at the platform docs and the passport doc for inspiration.